README.md

```diff
@@ -1,13 +1,21 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week we built a to-do app using React and redux-toolkit.
+We created new slices, new reducers and actions. And talked about what immutabilty means.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started out by doing a design sketch in Figma followed by some notes on each step the app should do.
+
+Then I started with selecting data from the store using useSelector and dispatching data to the store depending on the users actions.
+After that I created the components ToDoList that toogles the list and AddTask that lets the user do just that. When this mean functionalitys were in place I added header and also decided to use a Wrapper component to keep App clean.
+
+I wanted to spend some more time on styling this week and tried to follow my sketch to the minimum. I styled everything with styledcomponents, it all worked out nicely but could use a lesson on the optimal way of structuring it.
```

Comment:
+1. How CSS should be structured is still unclear and now with styled components it's even worse.